### PR TITLE
Fda 200 nsde

### PR DESCRIPTION
**Pull request by beardedfinch**

### CODE REVIEW

File: pages/data/maude/_meta.yaml [hunk 1]
```diff
@@ -1,38 +1,29 @@
 type: dataset
-documentTitle: 'openFDA › Datasets › MAUDE'
+documentTitle: 'openFDA › Datasets › NSDE'
```
Comment (dkrylovsb) [inline pages/data/maude/_meta.yaml]: NSDE in maude file?

File: pages/data/nsde/_meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+type: dataset
+documentTitle: 'openFDA › Datasets › MAUDE'
```
Comment (dkrylovsb) [inline pages/data/nsde/_meta.yaml]: MAUDE in nsde file.

Comment (beardedfinch) [inline pages/data/nsde/_meta.yaml]: Fixed. I apparently made the nsde changes in the maude dataset file. Both locations now appropriately reflect their datasets.

File: pages/data/maude/_meta.yaml (hunk 1 above)
Comment (beardedfinch) [inline pages/data/maude/_meta.yaml]: Fixed.

File: pages/data/_content.yaml [hunk 3]
```diff
@@ -1,25 +1,20 @@
--
-  url: /data/faers/
+- url: /data/faers/
   title: FAERS
   description:
-    - The FDA Adverse Event Reporting System (FAERS) is a database that contains information on adverse event and medication error reports submitted to FDA.
-    - See the [FAERS web site](http://www.fda.gov/Drugs/GuidanceComplianceRegulatoryInformation/Surveillance/AdverseDrugEffects/ucm082193.htm) for download information.
```
Comment (dkrylovsb) [inline pages/data/_content.yaml]: @beardedfinch Removal of this para intentional?

Comment (beardedfinch) [reply pages/data/_content.yaml]: Yup, I fixed some formatting on that data page.

**Pull request merged**